routing/index_graph_starter.cpp

```diff
@@ -135,12 +197,29 @@ void IndexGraphStarter::FindPointsWithCommonFeature(Joint::Id jointId0, Joint::I
   bool found = false;
   double minWeight = -1.0;
 
+  // Additional edges.
+  if (m_jointsOfAdditionalEdges.count(jointId0) != 0 && m_jointsOfAdditionalEdges.count(jointId1) != 0)
```

Comment:
> Why not to create a map from pair of joints to a list of directed edges? (or, to a directed edge if there can't be duplicates)

I need to have a quick way to check if a joint takes part in additional edges or not. For example in 
```c++
  template <typename F>
  void ForEachPoint(Joint::Id jointId, F && f) const
  {
...
    if (m_jointsOfAdditionalEdges.count(jointId) != 0)
    {
      for (DirectedEdge const & e : m_additionalZeroEdges)
...
```
or in
```c++
void IndexGraphStarter::GetAdditionalEdgesList(Joint::Id jointId, bool isOutgoing,
                                               vector<JointEdge> & edges)
{
  if (m_jointsOfAdditionalEdges.count(jointId) == 0)
    return;
...
```
If I had a map from a pair of joints to a directed edge I couldn't check quickly if a joint takes part in additional edges.